archs/amd64/text.xml

```diff
@@ -143,141 +143,66 @@ variable and <c><uri link="::eclass-reference/multilib.eclass/"/></c>).
 </subsection>
 
 <subsection>
-<title><c>Libdir</c> Links</title>
+<title>Libdir links</title>
 <body>
 
 <p>
-Currently, we provide several profiles, each with its own combination of <c>libdir</c>
-configurations.
+Currently, we provide several profiles, each with its own combination of
+<c>libdir</c> configurations. Table entries x86, amd64, etc. indicate that
+the directory contains objects for this ABI; entries with an arrow indicate
+a symlink to the respective directory.
 </p>
 
 <table>
   <tr>
-    <th>
-      Profile
-    </th>
-    <th>
-      lib32
-    </th>
-    <th>
-      lib
-    </th>
-    <th>
-      lib64
-    </th>
-  </tr>
-  <tr>
-    <ti>
-      2004.3
-    </ti>
-    <ti>
-      *l-&gt;emul*
-    </ti>
-    <ti>
-      d64
-    </ti>
-    <ti>
-      *l-&gt;lib*
-    </ti>
+    <th>Profile</th>
+    <th>lib</th>
+    <th>lib32</th>
+    <th>lib64</th>
+    <th>libx32</th>
   </tr>
   <tr>
-    <ti>
-      2004.3/lib64
-    </ti>
-    <ti>
-      *l-&gt;emul*
-    </ti>
-    <ti>
-      *l-&gt;64*
-    </ti>
-    <ti>
-      d64
-    </ti>
+    <ti>17.0</ti>
+    <ti>→ lib64</ti>
+    <ti>x86</ti>
+    <ti>amd64</ti>
+    <ti>inexistent</ti>
   </tr>
   <tr>
-    <ti>
-      &gt;=2005.0
-    </ti>
-    <ti>
-      d32
-    </ti>
-    <ti>
-      *l-&gt;64*
-    </ti>
-    <ti>
-      d64
-    </ti>
+    <ti>17.0/no-multilib</ti>
+    <ti>→ lib64</ti>
+    <ti>inexistent</ti>
+    <ti>amd64</ti>
+    <ti>inexistent</ti>
   </tr>
   <tr>
-    <ti>
-      &gt;=2005.0/no-multilib
-    </ti>
-    <ti>
-      d32
-    </ti>
-    <ti>
-      *l-&gt;64*
-    </ti>
-    <ti>
-      d64
-    </ti>
+    <ti>17.0/x32</ti>
+    <ti>→ libx32</ti>
+    <ti>x86</ti>
+    <ti>amd64</ti>
+    <ti>x32</ti>
   </tr>
   <tr>
-    <ti>
-      &gt;=2005.0/no-symlink
-    </ti>
-    <ti>
-      d32
-    </ti>
-    <ti>
-      d
-    </ti>
-    <ti>
-      d64
-    </ti>
+    <ti>17.1</ti>
+    <ti>x86</ti>
+    <ti>inexistent</ti>
+    <ti>amd64</ti>
+    <ti>inexistent</ti>
   </tr>
   <tr>
-    <ti>
-      &gt;=2005.0/no-symlink/no-lib32
-    </ti>
-    <ti>
-      inexistant
-    </ti>
-    <ti>
-      d32
-    </ti>
-    <ti>
-      d64
-    </ti>
+    <ti>17.1/no-multilib</ti>
+    <ti>n/a</ti>
+    <ti>inexistent</ti>
+    <ti>amd64</ti>
```

Comment:
nonexistent

Reply:
Hm, that word was there before, and Oxford online lists it: https://www.lexico.com/definition/inexistent

Anyway, updated.